src/shapes/circle.class.ts

```diff
@@ -1,232 +1,223 @@
+        'r="',
         this.radius,
-        degreesToRadians(this.startAngle),
-        degreesToRadians(this.endAngle),
-        false
-      );
-      this._renderPaintInOrder(ctx);
-    },
-
-    /**
-     * Returns horizontal radius of an object (according to how an object is scaled)
-     * @return {Number}
-     */
-    getRadiusX: function () {
-      return this.get('radius') * this.get('scaleX');
-    },
-
-    /**
-     * Returns vertical radius of an object (according to how an object is scaled)
-     * @return {Number}
-     */
-    getRadiusY: function () {
-      return this.get('radius') * this.get('scaleY');
-    },
-
-    /**
-     * Sets radius of an object (and updates width accordingly)
-     * @return {Circle} thisArg
-     */
-    setRadius: function (value) {
-      this.radius = value;
-      return this.set('width', value * 2).set('height', value * 2);
-    },
-
-    /**
-     * Returns object representation of an instance
-     * @param {Array} [propertiesToInclude] Any properties that you might want to additionally include in the output
-     * @return {Object} object representation of an instance
-     */
-    toObject: function (propertiesToInclude) {
-      return this.callSuper(
-        'toObject',
-        ['radius', 'startAngle', 'endAngle'].concat(propertiesToInclude)
-      );
-    },
-
-    /* _TO_SVG_START_ */
-
-    /**
-     * Returns svg representation of an instance
-     * @return {Array} an array of strings with the specific svg representation
-     * of the instance
-     */
-    _toSVG: function () {
-      const angle = (this.endAngle - this.startAngle) % 360;
-
-      if (angle === 0) {
-        return [
-          '<circle ',
-          'COMMON_PARTS',
-          'cx="0" cy="0" ',
-          'r="',
-          this.radius,
-          '" />\n',
-        ];
-      } else {
-        const { radius } = this;
-        const start = degreesToRadians(this.startAngle),
-          end = degreesToRadians(this.endAngle),
-          startX = fabric.util.cos(start) * radius,
-          startY = fabric.util.sin(start) * radius,
-          endX = fabric.util.cos(end) * radius,
-          endY = fabric.util.sin(end) * radius,
-          largeFlag = angle > 180 ? '1' : '0';
-        return [
-          '<path d="M ' + startX + ' ' + startY,
-          ' A ' + radius + ' ' + radius,
-          ' 0 ',
-          +largeFlag + ' 1',
-          ' ' + endX + ' ' + endY,
-          '" ',
-          'COMMON_PARTS',
-          ' />\n',
-        ];
-      }
-    },
-    /* _TO_SVG_END_ */
+        '" />\n',
+      ];
+    } else {
+      const { radius } = this;
+      const start = degreesToRadians(this.startAngle),
+        end = degreesToRadians(this.endAngle),
+        startX = cos(start) * radius,
+        startY = sin(start) * radius,
+        endX = cos(end) * radius,
+        endY = sin(end) * radius,
+        largeFlag = angle > 180 ? '1' : '0';
+      return [
+        '<path d="M ' + startX + ' ' + startY,
```

Comment:
```suggestion
        `<path d="M ${startX} ${startY}`,
```

Reply:
those we can change. Probably there will be a nicer way to get rid of those arrays in future too